packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/lib/simple.dart

```diff
@@ -35,6 +35,12 @@ class Model {
   final String value;
 }
 
+enum TestEnum {
+  one,
```

Comment:
We'd need a test which executes the a where/orderBy query and see if they work.
We'll also need an update operation.
And we need to check reads too